lib/logatron/logatron.rb

```diff
@@ -50,26 +50,25 @@ def debug(msg)
       logger.debug(msg)
     end
 
-    # Deprecated; use request_info/parse_request_info for transportation-layer neutral methods.
+    # @deprecated Use operation_context for transportation-layer neutral data.
     def http_headers
       {
         'X-Ascent-Log-Id' => msg_id,
         'X-Ascent-Site' => site
       }
     end
 
-    def request_info
+    def operation_context
       {
         message_id: msg_id,
         site: site
       }
     end
 
-    def parse_request_info(info)
-      if info
-        self.msg_id = info[:message_id] || info['message_id'] || self.msg_id
-        self.site = info[:site] || info['site'] || self.site
-      end
+    def operation_context=(info)
+      return unless info.is_a? Hash
```

Comment:
Do we really want to silently fail? My reason for commenting on the test was mainly that the description didn't match the test. Testing just for nil (and letting it blow up otherwise) would be fine too; just update the description text to match.